### ISSUE TITLE
redash alert: when claimed/unclaimed wallets drops below 50%

### ISSUE THREAD

**Issue by jenlij** [storj/storjscan#61]
Acceptance Criteria: 

- We should check for this every hour

**Comment by shaupt131** [comment]
Blocked by https://github.com/storj/storjscan/issues/62

**Comment by thepaul** [comment]
If you still want to do this in redash, I made some queries which can be made into alerts:

https://redash.datasci.storj.io/queries/3161/source [us1]
https://redash.datasci.storj.io/queries/3162/source [us2]
https://redash.datasci.storj.io/queries/3163/source [ap1]
https://redash.datasci.storj.io/queries/3164/source [eu1]
https://redash.datasci.storj.io/queries/3165/source [qa]

they all query BI dbs and refresh every 12h. But it's fine if you have a better way; I just wanted to braindump what I had found out about this ticket before letting it go entirely.

**Comment by joshdougall** [comment]
The following alerts have been created in Redash using [BI] datasources:

https://redash.datasci.storj.io/alerts/135 [us1]
https://redash.datasci.storj.io/alerts/136 [us2]
https://redash.datasci.storj.io/alerts/137 [ap1]
https://redash.datasci.storj.io/alerts/138 [eu1]
https://redash.datasci.storj.io/alerts/138 [qa]

If we want to change any of these, we can create a new ticket to migrate or upgrade monitoring.
